### ISSUE TITLE
Erroneous time played

### ISSUE THREAD

**Issue by o-schneider** [lutris/lutris#5277]
I noticed these last months that my time played recorded by lutris is way too high compared of the time I actually played the game. 

A few example, lutris says I played 44h42m at Rayman Origins, but I started playing this game one week ago (on 20/01) and it's certain that I didn't play 44h in one week between working and parenting. There's not even 44h of gameplay in this game. My real play time in the week should be around ~15h at most. Also, lutris says I played ~140h at Baldur's Gate 3 while my ingame save is at ~100h. I know there can be a discrepancy with the reloading and such, but 40h is too much. I also noticed this playing Quake (using a native source port) this summer, where a ~30 minutes sessions transformed into 2h+, so it affects wine and native games.

I didn't find anything relevant in the logs sadly. I have a post-exit command configured (to restart the compositor and screen saving configuration), I don't know if this can create such issues. I'd be happy to help more to debug this if I can.

**Comment by danieljohnson2** [comment]
Are these Steam games by any chance? We've recently added a feature to pick up play time from Steam.

**Comment by o-schneider** [comment]
No, these games are not steam games (GOG games actually). Steam games had their time synchronized with steam's counter as far as I remember, but I can't test right now as I've also the https://github.com/lutris/lutris/issues/5248 bug. And I'm aware of this issue since at least august, so since 0.5.13.

**Comment by danieljohnson2** [comment]
I've been thinking about this. I think the playtime timer has buggy timezone handling and could produce some crazy results if you have daylight savings time where you game; if you are gaming when the clock goes forward, it would count as extra playtime. I'll try to fix this.

But I still don't know where you can get 44 hours of extra playtime, so I don't think we've solved this.

**Comment by o-schneider** [comment]
Late follow-up on this, the issue I'm experiencing is definitely not related to timezone as it happened randomly over the last six months, but great if it lead you to a possible issue! I'm sorry to not have anything more specific, I'm trying to provoke the issue (with a little bit of imagination) to no avail for now, sadly.

**Comment by strycore** [comment]
Why would you ever think this is related to timezones????? ?????? (??)

The game hasn't quit properly and was left running in the background.

**Comment by danieljohnson2** [comment]
I didn't think that; I found a time-zone bug and fixed it, but it couldn't plausibly have caused this.

It's more plausible that the game didn't quit, but if it never quit, would Lutris ever write the updated playtime back?

Maybe it does quit, but only after a quite long delay; that might add up to 44 hours, I suppose.

**Comment by strycore** [comment]
The game obviously quit at some point but much latest than it should have. This has happened to me multiple time, getting over 20h of game play on games I've only played for a few minutes. 

This also includes putting your machine to sleep then resuming it.
